conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -97,6 +97,15 @@ object DynamicConformanceJob {
     processResult(result, performance, publishPath, stdPath)
   }
 
+  private def getExperimentalRuleEnabled()(implicit cmd: CmdConfig): Boolean = {
```

Comment:
this sound like this will make it enabled. But it actually returns if it is or is not enabled.